stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -492,6 +684,7 @@ class VhloToStablehloOpConverter : public OpConversionPattern<VhloOpTy> {
     //      mapping from VHLO to StableHLO.
     SmallVector<NamedAttribute> vhloAttrs = to_vector(vhloOp->getAttrs());
     SmallVector<NamedAttribute> stablehloAttrs;
+    if (failed(removeDefaults(*this, vhloOp, vhloAttrs))) return failure();
     if (failed(implodeSpecial(*this, vhloOp, vhloAttrs, stablehloAttrs)))
```

Comment:
Any way these two can impact each other? Can removeDefault make an implodeSpecial conversion invalid in any way? Or do imploded specials have default values that should be removed?